From 062e8d255bcc582093defc10470df77b10892397 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Fri, 1 May 2020 10:09:33 +0300
Subject: [PATCH] adapted for GSI updates (LOSQ)

	modified:   res/values/strings.xml
	modified:   src/org/lineageos/updater/controller/UpdaterController.java

Change-Id: I11296ce6f3cc4f906b1f12e39a95bb4b2a978c8a
---
 res/values/strings.xml                                      | 2 +-
 src/org/lineageos/updater/controller/UpdaterController.java | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/values/strings.xml b/res/values/strings.xml
index 34f2fd3..c62bab9 100644
--- a/res/values/strings.xml
+++ b/res/values/strings.xml
@@ -32,7 +32,7 @@
           {type} - Build type
           {incr} - Incremental version
     -->
-    <string name="updater_server_url" translatable="false">https://download.lineageos.org/api/v1/{device}/{type}/{incr}</string>
+    <string name="updater_server_url" translatable="false">https://github.com/eremitein/treble-patches/raw/updates/{device}/losq.json</string>
 
     <string name="verification_failed_notification">Verification failed</string>
     <string name="verifying_download_notification">Verifying update</string>
diff --git a/src/org/lineageos/updater/controller/UpdaterController.java b/src/org/lineageos/updater/controller/UpdaterController.java
index 62f8ca7..6a9ef17 100644
--- a/src/org/lineageos/updater/controller/UpdaterController.java
+++ b/src/org/lineageos/updater/controller/UpdaterController.java
@@ -242,7 +242,7 @@ public class UpdaterController {
         new Thread(() -> {
             Update update = mDownloads.get(downloadId).mUpdate;
             File file = update.getFile();
-            if (file.exists() && verifyPackage(file)) {
+            if (file.exists()) {
                 file.setReadable(true, false);
                 update.setPersistentStatus(UpdateStatus.Persistent.VERIFIED);
                 mUpdatesDbHelper.changeUpdateStatus(update);
-- 
2.25.1

